README.md

```diff
@@ -174,7 +218,10 @@ return [
     'http_url' => 'http://admin:changeit@localhost:2113',
     'streams' => ['quotes', 'accounts'],
     'group' => 'quote-email-senderer',
-    'namespace' => 'App\Events'
+    'namespace' => 'App\Events',
+    'event_to_class' => function ($event) {
```

Comment:
This should show how event name is converted to class name (and because it isn't required it should be in separate section; here, we should only have tcp_url, http_url, streams and group)

Reply:
Added sep section